packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -83,13 +84,17 @@ Limitations of JSON which Qwik solves:
 - JSON produces DAG. DAG stands for Directed Acyclic Graph, which means that the object which is being serialized can't have circular references. This is a big limitation because the application state is often circular. Qwik ensures that when the graph of objects gets serialized, the circular references get properly saved and then restored.
 - JSON can't serialize some object types. For example, DOM references, Dates, etc... Qwik serialization format ensures that such objects can correctly be serialized and restored. Here is a list of types that can be serialized with Qwik:
   - DOM references
-  - Dates (not yet implemented)
-  - Function closures (if wrapped in QRL).
+  - Promises
```

Comment:
well technically, promises are wrapped inside of "resources", so I think this should probably come with this disclaimer

Reply:
Thank you @shairez! I've added a link to the resources doc from this bit

Reply:
Thanks! I would add something like

Promises (wrapped in "Resources" which is the link)

It'll make it more explicit IMO

Reply:
My doubt is that although the standard is to wrap promises under resources, it is technically possible to have a promise serialized without it, as can be seen in the revised serialization example: https://qwik.builder.io/docs/cheat/serialization/, that works correctly.

What do you think? If it would serve as a way to encourage people to always use `useResource` for promises regardless, I can word it as you suggested. Let me know 🙂

Reply:
Let's bring in the big guns... @manucorporat what do you think?

Reply:
promises can be serialized in isolation, as long as they are resolved

Reply:
Thanks @manucorporat 
1. What do you mean by "in isolation"?
2. Do you think it's a good idea to have a disclaimer about it in this doc?

Reply:
i think it's ok! i meant that a promise can be serialized without involving a resource, but it's good we put a reference there, they are mostly used together

Reply:
cool thanks
merging